crawler/file_processing.py

```diff
@@ -758,7 +758,7 @@ def remove_bom(self, csvreader: DictReader) -> None:
 
             if has_bom:
```

Comment:
This could just be `if as_bytes_from_utf8[:3] == b"\xef\xbb\xbf":`?

Reply:
I think in this case I like declaring the variable has_bom, because it's unclear what `as_bytes_from_utf8[:3] == b"\xef\xbb\xbf"` actually means otherwise